src/pip/_internal/req/req_file.py

```diff
@@ -409,9 +409,7 @@ def parse_line(line):
 
         args_str, options_str = break_args_options(line)
 
-        # https://github.com/python/mypy/issues/1174
-        opts, _ = parser.parse_args(
-            shlex.split(options_str), defaults)  # type: ignore
+        opts, _ = parser.parse_args(shlex.split(options_str), defaults)
```

Comment:
Do you happen to know what `type: ignore` was actually for here? python/mypy#1174 is for variable reuse, but this does not seem to have anything to do with it.

Reply:
I'm actually not sure. Potentially related to a change in typeshed and the comment was mislabeled.